src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netcore.cs

```diff
@@ -9231,6 +10103,62 @@ public static Vector512<float> Invoke(Vector512<float> x)
 #endif
         }
 
+        private static Vector128<float> ElementWiseSelect(Vector128<float> mask, Vector128<float> left, Vector128<float> right)
+        {
+            if (Sse41.IsSupported)
+            {
+                return Sse41.BlendVariable(left, right, ~mask);
+            }
+            else return Vector128.ConditionalSelect(mask, left, right);
+        }
+
+        private static Vector128<int> ElementWiseSelect(Vector128<int> mask, Vector128<int> left, Vector128<int> right)
```

Comment:
Does this get inlined? If not, should we mark as aggressive inlining?

Reply:
Marked it as aggressive inlining.